### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by christianbmadsen**
Added more details about Braket and org repos to the readme.

*Issue #, if available:* No issues

*Description of changes:* Adding more details to the readme file

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,4 +1,28 @@
 # Welcome to the Amazon Braket GitHub organization
-Amazon Braket is a fully managed AWS service that helps researchers, scientists, and developers get started with quantum computing. Quantum computing has the potential to solve computational problems that are beyond the reach of classical computers because it harnesses the laws of quantum mechanics to process information in new ways.
+[Amazon Braket](https://aws.amazon.com/braket/) is a fully managed service on Amazon Web Services (AWS) helping researchers, scientists, and developers get started with quantum computing without having to worry setting up cloud infrastructure.
```
Comment (AbeCoull) [inline profile/README.md]:
```suggestion
[Amazon Braket](https://aws.amazon.com/braket/) is a fully managed service on Amazon Web Services (AWS) helping researchers, scientists, and developers get started with quantum computing without having to worry about setting up cloud infrastructure. 
```

File: profile/README.md [hunk 2]
```diff
@@ -1,4 +1,28 @@
...
+[Amazon Braket](https://aws.amazon.com/braket/) is a fully managed service on Amazon Web Services (AWS) helping researchers, scientists, and developers get started with quantum computing without having to worry setting up cloud infrastructure. 
 
-A number of frameworks and educational content has been opensourced that builds ontop of Amazon Braket. This Amazon Braket GitHub organization provides a good starting point to discover many of those opensourced artifacts.
\ No newline at end of file
+Quantum computing has the potential to solve computational problems beyond the reach of classical computers because it harnesses the laws of quantum mechanics to process information in new ways. 
+
+A number of open-source projects have been built ontop of Amazon Braket established. On this page you find starting point to discover these artifacts and how you can contribute to our quantum computing community!
```
Comment (AbeCoull) [inline profile/README.md]:
```suggestion
A number of open-source projects have been built ontop of Amazon Braket established. On this page you will find starting points to discover these artifacts and how you can contribute to our quantum computing community! 
```

File: profile/README.md [hunk 3]
```diff
@@ -1,4 +1,28 @@
...
 
-A number of frameworks and educational content has been opensourced that builds ontop of Amazon Braket. This Amazon Braket GitHub organization provides a good starting point to discover many of those opensourced artifacts.
\ No newline at end of file
+Quantum computing has the potential to solve computational problems beyond the reach of classical computers because it harnesses the laws of quantum mechanics to process information in new ways. 
+
+A number of open-source projects have been built ontop of Amazon Braket established. On this page you find starting point to discover these artifacts and how you can contribute to our quantum computing community! 
+
+# Amazon Braket fully supported repositories
+We aim to respond within few days for PRs and Issues raised in fully supported GitHub repositories. Customers can find these repositories installed on Amazon Braket Notebook Instances or on AWS managed containers for [Amazon Braket Hybrid Jobs](https://docs.aws.amazon.com/braket/latest/developerguide/braket-jobs-use.html).
```
Comment (AbeCoull) [inline profile/README.md]:
```suggestion
We aim to respond within a few days for PRs and Issues raised in fully supported GitHub repositories. Customers can find these repositories installed on Amazon Braket Notebook Instances or on AWS managed containers for [Amazon Braket Hybrid Jobs](https://docs.aws.amazon.com/braket/latest/developerguide/braket-jobs-use.html).
```

File: profile/README.md (hunk 2 above)
Comment (virajvchaudhari) [inline profile/README.md]:
```suggestion
A number of open-source projects have been built on top of Amazon Braket established. On this page you find starting point to discover these artifacts and how you can contribute to our quantum computing community! 
```

File: profile/README.md [hunk 4]
```diff
@@ -1,4 +1,28 @@
...
+
+* [Braket Python SDK](https://github.com/amazon-braket/amazon-braket-sdk-python) - Is a coding interface for quantum computing devices on Amazon Braket
```
Comment (AbeCoull) [inline profile/README.md]:
```suggestion
* [Braket Python SDK](https://github.com/amazon-braket/amazon-braket-sdk-python) - Provides a coding interface for quantum computing devices on Amazon Braket
```
nit: Keeps us from starting with a linking verb here

File: profile/README.md [hunk 5]
```diff
@@ -1,4 +1,28 @@
...
+* [Braket Examples](https://github.com/amazon-braket/amazon-braket-examples) - Contains tutorials on quantum computing with Amazon Braket
+* [PennyLane-Braket Plugin](https://github.com/amazon-braket/amazon-braket-pennylane-plugin-python) - Enables PennyLane code to run on Amazon Braket
+* [Qiskit-Braket Plugin](https://github.com/qiskit-community/qiskit-braket-provider) - Enables Qiskit code to run on Amazon Braket
+* [Braket Algorithm Library](https://github.com/amazon-braket/amazon-braket-algorithm-library/tree/main) - Contains community and first party implementations of quantum algorithms
+* [Braket Containers](https://github.com/amazon-braket/amazon-braket-containers/tree/main) - Offers a set of managed docker containers for hybrid jobs and good starting point for building custom containers
```
Comment (AbeCoull) [inline profile/README.md]:
```suggestion
* [Braket Containers](https://github.com/amazon-braket/amazon-braket-containers/tree/main) - Offers a set of managed docker containers for hybrid jobs and a starting point for building custom containers
```

File: profile/README.md [hunk 6]
```diff
@@ -1,4 +1,28 @@
...
+* [Braket Default Simulators](https://github.com/amazon-braket/amazon-braket-default-simulator-python) - provides an implementation of quantum simulators that you can run locally
```
Comment (AbeCoull) [inline profile/README.md]:
```suggestion
* [Braket Default Simulators](https://github.com/amazon-braket/amazon-braket-default-simulator-python) - Provides an implementation of quantum simulators that can run locally
```

File: profile/README.md [hunk 7]
```diff
@@ -1,4 +1,28 @@
...
+* [Braket Schemas](https://github.com/amazon-braket/amazon-braket-schemas-python) - Used to construct and verify payloads being sent and generated by Amazon Braket
```
Comment (AbeCoull) [inline profile/README.md]: nit: verb tense differs from the rest of the descriptions. Fine to keep but also could word:
```suggestion
* [Braket Schemas](https://github.com/amazon-braket/amazon-braket-schemas-python) - Constructs and verifies payloads being sent and generated by Amazon Braket
```

File: profile/README.md (hunk 4 above)
Comment (christianbmadsen) [reply profile/README.md]: Good call. Accepting change

File: profile/README.md [hunk 8]
```diff
@@ -1,4 +1,28 @@
...
+[Amazon Braket](https://aws.amazon.com/braket/) is a fully managed service on Amazon Web Services (AWS) helping researchers, scientists, and developers get started with quantum computing without having to worry about setting up cloud infrastructure. 
 
-A number of frameworks and educational content has been opensourced that builds ontop of Amazon Braket. This Amazon Braket GitHub organization provides a good starting point to discover many of those opensourced artifacts.
\ No newline at end of file
+Quantum computing has the potential to solve computational problems beyond the reach of classical computers because it harnesses the laws of quantum mechanics to process information in new ways. 
+
+A number of open-source projects have been built on top of Amazon Braket established. On this page you find starting point to discover these artifacts and how you can contribute to our quantum computing community!
```
Comment (AbeCoull) [inline profile/README.md]:
```suggestion
A number of open-source projects have been built on top of Amazon Braket. On this page you find starting point to discover these artifacts and how you can contribute to our quantum computing community! 
```

**Review by AbeCoull — APPROVED** [review]

**Pull request merged**